root/static/scripts/main/components/ConfirmSeedButtons.js

```diff
@@ -7,33 +7,50 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
+import * as React from 'react';
+
 import DBDefs from '../../common/DBDefs-client.mjs';
 
-const ConfirmSeedButtons = (): React$MixedElement => (
-  <>
-    <button type="submit">
-      {l('Continue')}
-    </button>
-    <button
-      className="negative"
-      onClick={() => {
-        if (history.length > 1) {
-          history.back();
-        } else {
-          window.location.replace(
-            window.location.protocol + '//' +
-            DBDefs.WEB_SERVER,
-          );
-        }
-      }}
-      type="button"
-    >
-      {l('Leave')}
-    </button>
-  </>
-);
+type PropsT = {
+  +autoSubmit: boolean,
+};
+
+const ConfirmSeedButtons = ({
+  autoSubmit,
+}: PropsT): React$MixedElement => {
+  const submitRef = React.useRef<HTMLButtonElement | null>(null);
+  React.useEffect(() => {
+    if (autoSubmit) {
+      submitRef.current?.click();
```

Comment:
@mwiencek, in the issue you mentioned using a delay before submitting the form. Please let me know if that's actually necessary.

Reply:
Thanks, I've added the dependency list and confirmed that this still works.

Reply:
Sorry, I forgot to update this PR after addressing this suggestion. I've created #3018 to pass the dependency list.